reporter/datadog_reporter.go

```diff
@@ -415,9 +401,11 @@ func (r *DatadogReporter) reportProfile(ctx context.Context) error {
 		return fmt.Errorf("failed to compress profile: %w", err)
 	}
 
-	if r.saveCPUProfile {
-		// write profile to cpu.pprof
-		f, err := os.Create("cpu.pprof")
+	if r.pprofPrefix != "" {
```

Comment:
Noting from Slack: https://dd.slack.com/archives/C06F2EP2SDA/p1730307153743729

This change is making the profiler output a copy of every pprof which was uploaded in the directory where the profiler is started, filling up the disk when the profiler is continuously ran in a workspace for example. What's the use case for making all the profiles available instead of keeping the last one?

Reply:
My intent was to use this as a debugging option, that is not meant to be enabled in production.
I removed it from docker-compose.yml